### PR TITLE
fix formatting

### PR DESCRIPTION
**Pull request by Xyueta**
it's does not equal to its

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <a href="https://github.com/couleur-tweak-tips/TweakList">
         <img src="https://img.shields.io/github/size/couleur-tweak-tips/TweakList/Master.ps1?label=Master%20script%20size&logo=PowerShell&logoColor=FFFFFF" alt="Code Size" />
     </a>
-    <a href="https://discord.com/invite/aPVMJy78Pa">
+    <a href="https://dsc.gg/ctt">
```
Comment (couleurm) [inline README.MD]: thanks for the nice formatting, could you revert that line though, this invite lets me know how much/who joined from clicking there 👍

Comment (Xyueta) [reply README.MD]: done

**Comment by couleurm** [comment]
thx <3

**Comment by Xyueta** [comment]
:)

**Pull request merged**